security/fma-superchainerc20.md

```diff
@@ -0,0 +1,116 @@
+
+- **Description**: For the `SuperchainTokenBridge` to validate cross-chain mints and burns correctly, the `SuperchainERC20` must appear at one consistent address on each chain when deployed. If a developer fails to deterministically deploy the token at the same address, the bridging logic cannot unify the token references across chains, leading to failed or incorrect cross-chain transfers.
+- **Risk Assessment**: Medium
+    - Potential impact: High. Without a consistent address, interoperability for the token is effectively disabled and `relayERC20` are likely to fail, causing a lost.
+    - Likelihood: Very Low. The interoperable set of chains follows the same opcode behavior and ensures identical availability of deployer contracts, such as `create2Deployer`.
+- **Mitigation**: For developers, employ the appropriate deterministic deployment tools, such as the one at `create2Deployer`. For users, refers to FM2 and FM3.
+- **Detection**: Deployers should test the deployment method used and verify contract addresses after a mainnet deployment.
+- **Recovery Path(s)**: Redeploy token contracts on a new address across chains and migrate userbase if it is needed.
+
+### FM5: Compromised Deployment Method
+
+- **Description**: The bridging logic assumes that if a `SuperchainERC20` contract address is identical across multiple chains, it refers to the expected code. However, if a deployer uses flawed or deployment methods that get compromised, a non-desired implementation may be deployed at the same address on one chain.
+- **Risk Assessment**: Medium
+    - Potential impact: High. In the worst-case scenario, compromised methods open the door to malicious behavior, e.g. where an attacker can occupy the address, and forge bridging events, and gain the ability to mint tokens in other chains.
+    - Likelihood: Low. This is possible if the deployer engages in poor practices or lacks opsec. Two examples of this are:
+        - Custom factories that utilize CREATE and share the same address are used, where tokens are deployed using the same bytecode and nonce.
+        - Proxy contracts where the admin is compromised or the proxy address front-runned.
+- **Mitigation**: For developers, employ the appropriate deterministic deployment tools, such as the one at `create2Deployer`. In the case of permissioned deployments, common for proxy contracts, ensure to maintain control over those deployments and ownership.
+- **Detection**: Support tickets filed by users reporting such issues.
+- **Recovery Path(s)**: Pause the token contract where possible and proceed to redeploy and migrate userbase if it is needed.
+
+## Action Items
+
+The following action items need to be done:
+
+- [ ]  Resolve all the comments.
+- [ ]  FM1: Implement tests for `SuperchainTokenBridge` predeploy. Currently, the L2 Genesis project is under development and it should take care of these tests.
+- [ ]  FM1: Run off-chain scripts and monitoring measurements (OP Labs).
```

Comment:
Adding some specifics here about monitoring would help with a definition of done